user-guide/Advanced_Modules/DOM/DOM_examples/DOM_BulkProcessing_Example.md

```diff
@@ -66,10 +86,10 @@ if (updateResult.HasFailures())
 }
 
 // Log what items were successfully removed.
-Log($"Could perform the update successfully for {updateResult.SuccessfulItems.Count} items");
+Log($"Could perform the update successfully for {updateResult?.SuccessfulItems.Count ?? 0} items");
```

Comment:
Feels a bit weird to need a null check on the updateResult here 🫤 Maybe we should add an if & check the tracedata on the CrudHelperComponent?

Reply:
That's in the next example that belongs to *Unexpected issue* (line 112). Just wanted this example to be correct. We could drop the other example and explain it here.

Reply:
Will combine the two examples.